<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>input</class>
 <widget class="QDialog" name="input">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mid_Number</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>34</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>181</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>NO.1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_mid_1">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>NO.2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_mid_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>NO.3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_mid_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>NO.4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_mid_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>NO.5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_mid_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>410</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tourist ID</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_settst">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>181</width>
     <height>76</height>
    </rect>
   </property>
   <property name="title">
    <string>Destination</string>
   </property>
   <widget class="QComboBox" name="comboBox_des">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>27</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>lmt</string>
   </property>
   <widget class="QSpinBox" name="spinBox_lmt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Mid_City</string>
   </property>
   <widget class="QRadioButton" name="radioButton_yes">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>27</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>YES</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_no">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>53</y>
      <width>43</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>NO</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Strategy</string>
   </property>
   <widget class="QRadioButton" name="radioButton_cost">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>27</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>minicost</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_time">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>53</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>minitime</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_both">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>79</y>
      <width>123</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>minicost_lmt</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_yes</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_mid_1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_yes</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_both</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_lmt</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
